### PR TITLE
Add address lookup analytics topic and consumer

### PR DESCRIPTION
**Pull request by matiit**

### CODE REVIEW

**Comment by uw-infra** [comment]
Received terraform plan request
```
Cluster: dev-aws
Module: pubsub/kafka-shared-msk-account-identity
Path: dev-aws/kafka-shared-msk/account-identity
Commit ID: 36f2ab9e598c75108271dffbe380e3f888871a27
Requested At: 2024-09-26T09:20:29Z
```
Do not edit this comment. This message will be updated once the plan run is completed.
To manually trigger plan again please post `@terraform-applier plan dev-aws/kafka-shared-msk/account-identity` as comment.

File: dev-aws/kafka-shared-msk/account-identity/address.tf [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import {
+  to = kafka_topic.account_identity_address_lookup_analytics_v1
+  id = "account-identity.address.lookup.analytics.v1"
+}
+
+resource "kafka_topic" "account_identity_address_lookup_analytics_v1" {
+  config = {
+
+    "compression.type" = "zstd"
```
Comment (sbuliarca) [inline dev-aws/kafka-shared-msk/account-identity/address.tf:9]: It would be good to define remote storage too.

**Comment by uw-infra** [comment]
Received terraform plan request
```
Cluster: dev-aws
Module: pubsub/kafka-shared-msk-account-identity
Path: dev-aws/kafka-shared-msk/account-identity
Commit ID: 9d645c0a674185c0bd33ae6fb58c0c059dd625d1
Requested At: 2024-09-26T09:23:15Z
```
Do not edit this comment. This message will be updated once the plan run is completed.
To manually trigger plan again please post `@terraform-applier plan dev-aws/kafka-shared-msk/account-identity` as comment.

**Comment by uw-infra** [comment]
Received terraform plan request
```
Cluster: dev-aws
Module: pubsub/kafka-shared-msk-account-identity
Path: dev-aws/kafka-shared-msk/account-identity
Commit ID: 06fce6f630b88d6f260c93e856928b1c2b26358b
Requested At: 2024-09-26T09:23:53Z
```
Do not edit this comment. This message will be updated once the plan run is completed.
To manually trigger plan again please post `@terraform-applier plan dev-aws/kafka-shared-msk/account-identity` as comment.

**Review by sbuliarca — APPROVED** [review]

**Pull request merged**